.github/workflows/release.yml

```diff
@@ -120,14 +118,17 @@ jobs:
       - name: Install GDAL
         env:
           VCPKG_DEFAULT_TRIPLET: ${{ matrix.triplet }}
+
         run: |
-          vcpkg install gdal[core] --overlay-triplets=./ci/custom-triplets
+          echo "VCPKG_INSTALLATION_ROOT: ${VCPKG_INSTALLATION_ROOT}"
```

Comment:
Is this a debugging left-over? (the variable is documented by github actions about what it is)